documentation/frontend/reference/testing_guidelines.md

```diff
@@ -357,15 +357,43 @@ output of failed tests by finding the `trace.zip` for the relevant test in the
 [Playwright Trace Viewer](https://playwright.dev/docs/trace-viewer) to inspect
 these (or open the zip yourself and poke around the files on your own).
 
-For Playwright tests failing in CI, a GitHub comment will appear with a link to
-download an artifact of the `test-results` folder.
+Playwright tests in CI are run with `-u` option by default, this means that
+snapshots will automatically be updated for modified parts of the UI if
+Playwright detects that. See
+[Updating snapshots](/frontend/guides/test.md#updating-snapshots) for more
+reading about this.
+
+When this happens, a GitHub comment will appear with a link to download zipped
+artifacts named in the form `*_snapshot_diff.zip`. Once downloaded, they can be
+decompressed and applied to your working branch by running:
+
+`ov unzip -p *_snapshot_diff.zip | git apply`
+
+The above command basically uses the `unzip` tool to unpack the contents of the
+downloaded archive file named `*_snapshot_diff.zip`, the `-p` option prevents it
+from actually creating any extracted file but rather prints the contents to the
+standard output which is then piped to `git apply`.
+
+```{note}
+For the above command to work verbatim for you, make sure to have saved the downloaded artifact to the root of the project from where you're able to run git commands. Otherwise, you'd need to adjust the path to the downloaded file in the command accordingly. Should you settle for the former, keep in mind that you'd also need to delete the file(`*_snapshot_diff.zip`) after successfully applying the changes to avoid committing them.
```

Comment:
Unfortunately #1986 means we need to manually wrap the text inside code blocks, to at least _around_ 80 characters. It's so tedious I really don't try too hard other than to get it around 80. Still needs to be done, unfortunately.

There's also a space missing between `file` and the parenthetical after it.

Reply:
Also: for the files to be in context for `ov`, they _must_ be downloaded to the repository. The repository is the only host filesystem location that ov explicitly mounts into the container. Probably worth just adding that to the instructions: download the zipped snapshot patches to the repository root, etc.